x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/import_rules_route.ts

```diff
@@ -57,10 +55,7 @@ export const importRulesRoute = (router: IRouter, config: ConfigType, ml: SetupP
         query: buildRouteValidation<typeof importRulesQuerySchema, ImportRulesQuerySchemaDecoded>(
           importRulesQuerySchema
         ),
-        body: buildRouteValidation<
-          typeof importRulesPayloadSchema,
-          ImportRulesPayloadSchemaDecoded
-        >(importRulesPayloadSchema),
+        body: schema.any(), // validation on file object is accomplished later in the handler.
```

Comment:
This will work, I am going to assume there is no clean way to just tell it validation off but please still give me the body as is. I still approve of this as a valid way of doing this though and appreciate the comment you put in there for future maintainers.